src/pages/MonthlyGood/HowItWorksOptionalChoiceExp.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
@egoroza @vijayabhandari Please make a follow up ticket to move this to Contentful if we go forward with these after the experiment.

Alternatively, Emily and I could work on a much smaller component wrapper to just load this content from Contentful before launch too.

Reply:
would be good to have this loaded from contentful regardless of whether or not we go through with this experiment imo. it's hardcoded for the control/current experience as well

Reply:
Agree! But we should probably set out a separate task to do that work.

Reply:
created https://kiva.atlassian.net/browse/CORE-543